HISTORY.rst

```diff
@@ -2,9 +2,50 @@
 History
 =======
 
-3.26.0 (2021-01-08)
+3.27.0 (2021-01-21)
 -------------------
 
+Changes:
+
+- Upgrades including Snowfakery 1.4 with 
+  min, max, round functions. PR #2335
+
+- The ensure_record_types task has a new option, force_create,
+  which will create the Record Type even if other Record Types
+  already exist on the object. PR #2323
```

Comment:
```suggestion
- The ``ensure_record_types`` task has a new option, ``force_create``,
  which will create the Record Type even if other Record Types
  already exist on the object. PR #2323
```

Task and option names should be double-backticked to render in a fixed-width font. There are others to adjust below...